lib/RateLimiterStoreAbstract.js

```diff
@@ -284,6 +284,7 @@ module.exports = class RateLimiterStoreAbstract extends RateLimiterAbstract {
     return new Promise((resolve, reject) => {
       this._delete(rlKey, options)
         .then((res) => {
+          this._inmemoryBlockedKeys.delete(key);
```

Comment:
It seems like `delete` function is not deleting in-memory data?

Reply:
@evan361425 Good catch. It should be `this._inmemoryBlockedKeys.delete(rlKey);` (var is `rlKey` - prepared by specific function for keys)